### ISSUE TITLE
Error: Undefined name 'ClosedCaptionFormat'.

### ISSUE THREAD

**Issue by zxl777** [Hexer10/youtube_explode_dart#86]
Error: Undefined name 'ClosedCaptionFormat'.

```
var trackInfo = manifest.getByLanguage('en',
                      format: ClosedCaptionFormat.srv3);
```

**Comment by Hexer10** [comment]
Which version are you using? I cannot reproduce the error, and also how do you import the library?

**Comment by zxl777** [comment]
youtube_explode_dart: ^1.7.1

import 'package:youtube_explode_dart/youtube_explode_dart.dart';

in flutter app

**Comment by zxl777** [comment]
Can you provide a complete Flutter Example that can cover all APIs, so that it is convenient to reproduce and find problems.

**Comment by Hexer10** [comment]
I've published v1.7.2 which should address this issue.

I'll cover more APIs in the flutter example later on, as I do not have time at moment, PRs are always welcome

**Comment by zxl777** [comment]
Thank you, I also hope to have the opportunity to submit some PRs.
